<?xml version="1.0" ?>
<launch>
  <include file="$(find franka_control)/launch/franka_control.launch" pass_all_args="true"/>
  <arg name="arm_id" default="fr3" />
  <rosparam command="load" file="$(find fr3_ros)/config/fr3_ros.yaml" subst_value="true" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="waypoint_cbf_controller"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find fr3_ros)/launch/robot.rviz"/>
</launch>